helm_deploy/hmpps-incentives-api/values.yaml

```diff
@@ -49,22 +49,6 @@ generic-service:
       SPRING_R2DBC_PASSWORD: "database_password"
       DATABASE_NAME: "database_name"
       DATABASE_ENDPOINT: "rds_instance_endpoint"
-    hmpps-domain-events-topic:
```

Comment:
Moved into each environment's config, will revert once we're happy to make same changes in PreProd & Prod